<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Header</class>
 <widget class="QWidget" name="Header">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(51,153,255);
}

QLabel
{
	border : 1px solid black;
	font:15px, &quot;SimHei&quot;;
}</string>
  </property>
  <widget class="QLabel" name="label_CarNumber">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:25px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Title">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>450</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:25px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Time">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>0</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Speed">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>140</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelVoltage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:25px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_icon">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>511</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelDrag">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>32</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/tracking.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelRestrictSpeed">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>32</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:20px ,&quot;SimHei&quot;;
background-color: rgb(255, 255, 0);</string>
   </property>
   <property name="text">
    <string>限速40</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelWashMode">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>32</y>
     <width>90</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:20px ,&quot;SimHei&quot;;
</string>
   </property>
   <property name="text">
    <string>洗车模式</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelFault">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>32</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/fault.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelConnFault">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>601</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color:red;
font:50px &quot;SimHei&quot;;</string>
   </property>
   <property name="text">
    <string>通信中断</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelFaultComfirm">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>801</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;
color:red;
font:30px &quot;SimHei&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>30</y>
     <width>81</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">outline:none;
border-image: url(:/image/buttonUP.jpg);
font:20px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirmAll">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>81</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">outline:none;
border-image: url(:/image/buttonUP.jpg);
font:20px, &quot;SimHei&quot;;
color:rgb(0, 32, 96);
border-radius:2px;</string>
   </property>
   <property name="text">
    <string>确认全部</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMend">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>32</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/maintain.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTractionActive">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>32</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/motorAllow.gif);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelTalkToPassenger">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>32</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/passageTalk.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelBrake">
   <property name="geometry">
    <rect>
     <x>295</x>
     <y>32</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color:white;
font:25px;</string>
   </property>
   <property name="text">
    <string>EB</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_CarNumber</zorder>
  <zorder>label_Title</zorder>
  <zorder>label_Time</zorder>
  <zorder>label_Speed</zorder>
  <zorder>labelVoltage</zorder>
  <zorder>label_icon</zorder>
  <zorder>labelDrag</zorder>
  <zorder>labelRestrictSpeed</zorder>
  <zorder>labelWashMode</zorder>
  <zorder>labelFault</zorder>
  <zorder>labelMend</zorder>
  <zorder>labelTractionActive</zorder>
  <zorder>labelTalkToPassenger</zorder>
  <zorder>labelBrake</zorder>
  <zorder>labelFaultComfirm</zorder>
  <zorder>btnConfirmAll</zorder>
  <zorder>btnConfirm</zorder>
  <zorder>labelConnFault</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
